eip-draft_offchain_checks_erc1155.md

```diff
@@ -0,0 +1,265 @@
+---
+eip: 
+title: Off-chain Checks
```

Comment:
#7099 shares the same title, consider distinguishing between the two by adding the token to the title.

e.g. 
Offchain checks for ERC-20 distribution
Offchain checks for ERC-1155 distribution